js/search/controllers/search.controller.js

```diff
@@ -24,14 +24,42 @@
      * @ngdoc controller
      * @name jwShowcase.search.SearchController
      */
-    SearchController.$inject = ['$state', 'platform', 'searchFeed'];
-    function SearchController ($state, platform, searchFeed) {
+    SearchController.$inject = ['$scope', '$state', '$stateParams', 'platform', 'searchFeed', 'api'];
+    function SearchController ($scope, $state, $stateParams, platform, searchFeed, api) {
 
         var vm = this;
 
-        vm.feed = searchFeed;
+        vm.cardClickHandler     = cardClickHandler;
+        vm.showMoreClickHandler = showMoreClickHandler;
+        vm.itemsLeft            = itemsLeft;
 
-        vm.cardClickHandler = cardClickHandler;
+        vm.feed = null;
+
+        var limit = 10;
+
+        if($stateParams.searchInCaptions) {
+            var partFeed = angular.copy(searchFeed);
+
+            partFeed.playlist = searchFeed.playlist.slice(0, limit);
+
+            addItemsToFeed(partFeed.playlist).then(function () {
+                vm.feed = partFeed;
+
+                $scope.$apply();
+            });
+        } else {
+            vm.feed = searchFeed;
+        }
+
+        function addItemsToFeed(items) {
```

Comment:
Space between function name and arguments

Reply:
Agree